scripts/can_bundle.py

```diff
@@ -79,23 +79,25 @@ def main(argv):
 def fetch_bundle(source: str, bundle_uuid: str, bundle_version: str) -> Bundle:
     for catalog in config.catalogs:
         plugin = plugin_for(catalog)
-        sources = set(map(str, plugin.sources))
         try:
             parsed_source = plugin.resolve_source(source)
-        except Exception:
-            pass
+        except Exception as e:
+            logger.debug('Skipping catalog %r (incompatible source)', catalog)
```

Comment:
Since you're touching this

```suggestion
            log.debug('Skipping catalog %r (incompatible source)', catalog)
```